WordPress/Classes/Utility/BackgroundTasks/WeeklyRoundupBackgroundTask.swift

```diff
@@ -538,21 +537,20 @@ class WeeklyRoundupNotificationScheduler {
         comments: Int,
         likes: Int,
         periodEndDate: Date,
-        context: NSManagedObjectContext,
-        completion: @escaping (Result<Void, Error>) -> Void) {
-
-            var siteTitle: String?
-            var dotComID: Int?
-
-            context.performAndWait {
-                dotComID = site.dotComID?.intValue
-                siteTitle = site.title
-            }
+        completion: @escaping (Result<Void, Error>) -> Void
+    ) {
+        var siteTitle: String?
+        var dotComID: Int?
+
+        site.managedObjectContext?.performAndWait {
+            siteTitle = site.title
+            dotComID = site.dotComID?.intValue
+        }
 
-            guard let dotComID = dotComID else {
-                // Error
-                return
-            }
+        guard let dotComID = dotComID else {
+            // Error
+            return
```

Comment:
Given [this code should be impossible to reach](https://github.com/wordpress-mobile/WordPress-iOS/pull/19422/files/bd2f325ccf7a029b708bab420e44b40c9df51941#r991719539), what do you think of leaving a note about it?

```suggestion
            // This code path should be impossible to reach.
            // At the time of writing, there is code to filter non-WordPress.com sites before calling this method.
            return
```

Alternatively, would this count as "developer error"?

```suggestion
        fatalError("Inconsistent State: Non-WordPress.com sites should have been filtered before calling this method.")
```

Reply:
🤔 this is how I see [the comment I just posted](https://github.com/wordpress-mobile/WordPress-iOS/pull/19422/files/bd2f325ccf7a029b708bab420e44b40c9df51941#r991765499) 😓 

![image](https://user-images.githubusercontent.com/1218433/194985187-f39cb20f-877f-4c64-b4c9-d377c9429806.png)

If that's how you see it, too, you should be able to see my suggestions by looking at the comment source via the edit button. But I hope that's just a temporary hiccup.

Reply:
The suggestion blocks look normal on my side 😸 

I've added a `fatalError` in https://github.com/wordpress-mobile/WordPress-iOS/pull/19422/commits/f095331ccb7cbc84f09e8b176990fd6c75666e09